Sources/AblyAssetTrackingSubscriber/DefaultSubscriber.swift

```diff
@@ -37,8 +37,19 @@ class DefaultSubscriber: Subscriber {
         trackableId: String,
         resolution: Resolution?,
         logHandler: InternalLogHandler?) {
-            
-        self.workingQueue = DispatchQueue(label: "com.ably.Subscriber.DefaultSubscriber", qos: .default)
+
+        self.workerQueue = WorkerQueue(
+            properties: SubscriberWorkerQueueProperties(),
+            workingQueue: DispatchQueue(label: "com.ably.Subscriber.DefaultSubscriber", qos: .default),
+            logHandler: nil,
```

Comment:
Can we move the `self.logHandler =` line above this and pass it to the worker queue?